generate-component/Template.test.tsx

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import { screen, render } from "@testing-library/react";
+import Template from "./Template";
+
+describe("Template", () => {
```

Comment:
The only problem with the "read file" from file approach is that I don't think I have a good way of forcing the describe block's `Template` to be lowercased.

I think leaving it like this is fine and adding a comment to handle it is worth the benefits of having the ts linted file.